hazelcast-sql/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -321,6 +356,27 @@ SqlDrop SqlDropMapping(Span span, boolean replace) :
     }
 }
 
+/**
+ * Parses DROP DATA LINK statement.
+ */
+SqlDrop SqlDropDataLink(Span span, boolean replace) :
+{
+    SqlParserPos pos = span.pos();
+
+    SqlIdentifier name;
+    boolean ifExists = false;
+}
+{
+    <DATA> <LINK>
```

Comment:
in design doc it is defined as `DROP DATA LINK <name> [IF EXISTS];` - `IF EXISTS` at the end

Reply:
Then, it's wrong in design doc - `<CREATE/DROP> ... <IF [NOT] EXISTS> name` is more common.

Reply:
updated